packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -349,6 +349,24 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
           }
           return;
 
+        case KeyCodes.tab:
```

Comment:
`FocusZone` *is* missing a "tab-only" mode, but it would be more appropriate to tie that to something like `FocusZoneDirection.natural`, or a similar opt-in behavior.
The point of `FocusZone` is explicitly to have the tab key enter and exit the entire zone, not move items within the zone.

Reply:
Done. From our conversation earlier, I added a flag to only allow tabbing if we override the tabbing feature.